service/frontend/wrappers/metered/metered_test.go

```diff
@@ -76,7 +76,10 @@ func TestSignalMetricHasSignalName(t *testing.T) {
 	signalRequest := &types.SignalWorkflowExecutionRequest{
 		SignalName: "test_signal",
 	}
-	handler.SignalWorkflowExecution(context.Background(), signalRequest)
+	err := handler.SignalWorkflowExecution(context.Background(), signalRequest)
+	if err != nil {
+		return
```

Comment:
did you mean to return here on error or assert no err?

Reply:
I think my copilot added this accidentally via that `handle error` thing. removing.